pkg/controller/controller.go

```diff
@@ -232,9 +232,9 @@ func worker(queue workqueue.RateLimitingInterface, resourceType string, maxRetri
 // places so this method fetches the Service Class, Service Plan and creates
 // a brokerClient to use for that method given an ServiceInstance.
 func (c *controller) getServiceClassPlanAndServiceBroker(instance *v1alpha1.ServiceInstance) (*v1alpha1.ServiceClass, *v1alpha1.ServicePlan, string, osb.Client, error) {
```

Comment:
Could we document in this comment the side effect of setting the [Class/Plan]Ref if it's currently unset?

Reply:
done.